control-plane/cmd/webhook-kafka/main.go

```diff
@@ -77,6 +84,35 @@ func NewDefaultingAdmissionController(ctx context.Context, _ configmap.Watcher)
 	)
 }
 
+func NewPodDefaultingAdmissionController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
+
+	// A function that infuses the context passed to Validate/SetDefaults with custom metadata.
+	ctxFunc := func(ctx context.Context) context.Context {
+		return ctx
+	}
+
+	return defaulting.NewAdmissionController(ctx,
+		// Name of the resource webhook.
+		"pods.defaulting.webhook.kafka.eventing.knative.dev",
+
+		// The path on which to serve the webhook.
+		"/pods-defaulting",
+
+		// The resources to default.
+		// We use only defaulting callbacks for pods.
+		map[schema.GroupVersionKind]resourcesemantics.GenericCRD{},
```

Comment:
Do we add Pod{} type to this map here?

Reply:
No, this is done by `defaultingCallbacks` which is from `Pod` -> `custom function`.

Reply:
Ok. Thanks!